spec/models/family_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Family do
+  describe "validations" do
+    subject { Family.new(last_name: nil) }
+
+    context "when creating a new family with no last_name is not valid" do
+      it { should_not allow_value(nil).for(:last_name)}
+    end
+
+    context "when creating a new family with no capability is not valid" do
+      it { should_not allow_value(nil).for(:capability)}
+    end
+
+    context "when creating a new family that is valid it is saved" do
+      it { expect(build(:family)).to be_valid }
+    end
+  end
+end
```

Comment:
The shoulda matcher tests that Elle suggested would work here:
line 5: subject { build(:family) }
line 8: it { is_expected.to validate_presence_of(:last_name) }
line 12 it { is_expected.to validate_presence_of(:capability) }
line 16 I don't think you need this test as you seem to be testing that the factory works

Reply:
Great work, I am intrigued to see more of your project, you have a lot to cover!